### DISCUSSION TITLE
Can watch group by 32bits/4 bytes?

### DISCUSSION THREAD

**Discussion by remy** [maziac/DeZog#54] [Q&A] [answered]
I'm using the watch to quickly inspect a spot in memory, but the value I want is 32bit. I've used watch to render as 16bit but I either I'm doing it wrong, or 32bit goes very wrong:

<img width="359" alt="Screenshot 2021-04-25 at 10 07 46" src="https://user-images.githubusercontent.com/13700/115987705-19c27900-a5ae-11eb-9fcc-9fbd776db459.png">

Is this something that's available or is it just not supported? (I didn't want to raise a bug if it wasn't actually a bug!)

**Comment by maziac** [comment]
Actually: it is a bug. 
Never tested something different than 1 or 2 so far.
Could you enter it as bug?

**Comment by maziac** [comment, marked as answer]
Was fixed meanwhile.
